indexer-app/src/cmr/indexer/data/concepts/collection/collection_util.clj

```diff
@@ -10,3 +10,12 @@
   (try
     (str (Integer/parseInt version-id))
     (catch Exception _ version-id)))
+
+(defn opendap-url?
+  "Determines if the related-url is a OPeNDAP service url."
+  [related-url]
+  (if (and (= (:URLContentType related-url)  "DistributionURL")
```

Comment:
Do we have these magic strings stored somewhere in umm-spec-lib?

I do see some other places in the code where we try to determine if something is an OPeNDAP type - might be useful to consolidate OPeNDAP checks in one namespace.

```
opendap-type (when (= code "GET DATA : OPENDAP DATA (DODS)")
                            "USE SERVICE API")
```
```
  :Subtype (if opendap-type
                "OPENDAP DATA (DODS)"
                (or (:Subtype types-and-desc) (:Subtype service-url)))
```

Reply:
Nope, @eereiter said they probably won't change this year.

Reply:
I'll throw them somewhere.